### PR TITLE
teams/tidb: a vote for joccau as tidb committer

### PR DESCRIPTION
**Pull request by 3pointer**
as title.

### CODE REVIEW

**Review by YuJuncen — APPROVED** [review]
LGTM

File: teams/tidb/membership.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
         "hawkingrei",
         "ichn-hu",
         "imtbkcat",
+	"joccau",
```
Comment (kennytm) [inline teams/tidb/membership.json:63]:
```suggestion
        "joccau",
```

**Review by kennytm — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

**Comment by YuJuncen** [comment]
@3pointer Maybe we need to keep it open for at least 3 days?

**Pull request merged**